lib/teselagen/utils/crispr.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+from typing import List, Optional
+from dna_features_viewer import GraphicFeature, GraphicRecord
+
+
+def show_crispr_grna_results(sequence:str, guides: List[dict], indexes: Optional[List[int]]=None, scoreField:str="onTargetScore"):
```

Comment:
Should we include a `scoreField` description in docstring?

Reply:
Done!